exercises/practice/accumulate/.meta/config.json

```
@@ -0,0 +1,22 @@
+{
+  "blurb": "Implement the `accumulate` operation, which, given a collection and an operation to perform on each element of the collection, returns a new collection containing the result of applying that operation to each element of the input collection.",
+  "authors": [
+    "Kryo"
+  ],
+  "contributors": [
+    "Kryo"
+  ],
```

Someone is either an author or a contributor, but not both. You're definitely the author :)

```suggestion
  "contributors": [],
```